nemo_aligner/data/nlp/datasets.py

```diff
@@ -344,16 +348,63 @@ def encode(self, text, append_eod=False):
 
         return text_ids, len(text_ids)
 
+    def _convert_messages(self, input_list):  # TODO: (@adithyare) this method should live elsewhare..
```

Comment:
Function seems pure so would be good to decorate with `@staticmethod`. This also means it'd be nice to add a unit test for this.

I'm not sure what this is converting to and from, could you add a docstring or give the function a name informing what we are converting to and from?